tests/parsers/pipe.parser.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('PipeParser', () => {
 	});
 
 	it('should extract strings with escaped quotes', () => {
-		const contents = `Hello {{ 'World\'s largest potato' | translate }}`;
+		const contents = `Hello {{ 'World\\'s largest potato' | translate }}`;
```

Comment:
> Single backslash + single quote, is the correct way to perform the escaping in templates

I may be wrong, but I think that the double escape is necessary: if you try to write
`Hello {{ 'World\'s largest potato' | translate }}`
in a HTML template, you'll get an error "Uncaught error: unexpected identifier"

if you use the double backslash, it will be displayed as expected

note also that if you add
`console.log(contents)`
just after the declaration, and see that the slash has disappeared if you put only one
you see 1 slash if you put 2